conanfile.py

```diff
@@ -101,6 +110,10 @@ def build_requirements(self):
         if self.options.build_docs:
             self.build_requires("doxygen/1.9.4") # doxygen/1.9.5 will update dependency on zlib/1.2.12 to zlib/1.2.13
 
+        if self.options.get_safe("link_with_mold", False):
```

Comment:
Because we may have deleted this option, we now access it with the `get_safe` function where we provide a default, in this case false so it is unsupported, when we access the value as detailed here: https://docs.conan.io/2/reference/conanfile/attributes.html#options